src/org/team4909/preseason2018/subsystems/SampleVelocitySubsystem.java

```diff
@@ -0,0 +1,17 @@
+package org.team4909.preseason2018.subsystems;
+
+import org.team4909.bionic.utils.setpoints.VelocitySetpoint;
+import org.team4909.bionic.utils.subsystems.BionicVelocitySubsystem;
+import edu.wpi.first.wpilibj.SpeedController;
+
+public class SampleVelocitySubsystem extends BionicVelocitySubsystem {
+	public VelocitySetpoint  VelocitySetpoint  = new VelocitySetpoint(this, 1.0);
```

Comment:
you shouldn't use the same name for the class itself and the variable name. 

Ex:
`public VelocitySetpoint velocitySetpoint`